packages/app/cypress/e2e/create-spec.cy.ts

```diff
@@ -0,0 +1,40 @@
+import type { e2eProjectDirs } from '@packages/frontend-shared/cypress/e2e/support/e2eProjectDirs'
+
+function scaffoldAndVisit(project: typeof e2eProjectDirs[number], type: 'component' | 'e2e') {
```

Comment:
we need these tests for every combo of project, most likely, like nextjs, etc etc.